lib/dai-plugin-mcd/src/CollateralService.js

```diff
@@ -0,0 +1,160 @@
+import Maker from '@makerdao/dai';
+import { WAD, RAY } from '@makerdao/dai/utils/constants';
+import BigNumber from 'bignumber.js';
+import { getIlkForCurrency, getSpotContractNameForCurrency, getPipContractNameForCurrency, getFlipContractNameForCurrency } from './utils';
```

Comment:
make sure you're running the linter step when you make commits so that long lines get formatted correctly, etc. by prettier... is husky not working for you?

Reply:
prettier wasn't checking files in the `lib` folder, so I updated the prettier command to include it